misc/ls-extensions/modules/json-to-record-converter/build.gradle

```diff
@@ -36,6 +36,7 @@ dependencies {
         exclude group: "javax.validation", module: "validation-api"
         exclude group: 'com.google.guava', module: 'guava'
     }
+    implementation 'com.google.guava:guava:"${project.guavaVersion}"' // Required during runtime
```

Comment:
I tried this syntax for dependency configuring but was failing. The syntax worked for me was 

implementation "com.google.guava:guava:${project.guavaVersion}"  (even using single quotes instead of double quotes was not working)

Reply:
Yes you need to use double quotes for String templates as `${project.guavaVersion}` is being used